components/Popover/Popover.stories.tsx

```diff
@@ -62,3 +65,39 @@ export const RichContent: ComponentStory<typeof PopoverForStory> = (args) => (
     </PopoverForStory>
   </Container>
 );
+
+export const IconTrigger: ComponentStory<typeof PopoverForStory> = (args) => (
+  <Container>
+    <PopoverForStory {...args}>
+      <PopoverTrigger asChild>
+        <Button size="small">
+          <HamburgerMenuIcon />
+        </Button>
+      </PopoverTrigger>
+      <PopoverContent arrowCss={{ color: '$primary' }} css={{ bc: '$primary', p: '$2' }}>
+        <Text css={{ c: 'white' }}>Content</Text>
+      </PopoverContent>
+    </PopoverForStory>
+  </Container>
+);
+
+export const RowAnchor: ComponentStory<typeof PopoverForStory> = (args) => (
+  <Container>
+    <PopoverForStory {...args}>
+      <PopoverAnchor asChild>
+        <Box css={{ bc: '$orange4', p: '$2', display: 'inline-block' }}>
```

Comment:
I feel we could improve this Story by vertically centralizing the text, and moving it outside the `PopoverForStory` then the Popover will be centralized to the `HamburgerMenuIcon`. 
I also would remove this orange and let the text in `$hiContrast`. WDYT?

This would be the result:

https://user-images.githubusercontent.com/38889364/158204340-63f25483-80bb-4fc8-9916-910c82dfe5e5.mov

Reply:
@paulocfjunior This example is here to demonstrate the usage of `PopoverAnchor` to anchor the popover to something else than the trigger

Reply:
Humm got it, sounds good.

I'd just keep my recommendation for the text color then, to make it more legible in both color schemes.

Reply:
And it's optional IMO, so I'm approving the PR already.

Reply:
done